Anonymize/Anonymizer.php

```diff
@@ -22,38 +26,53 @@
 class Anonymizer
 {
     /**
-     * The annotation reader service
-     *
      * @var AnnotationReader
      */
     private $annotationReader;
 
+    /**
+     * @var PropertyAnonymizer
+     */
+    private $propertyAnonymizer;
+
     /**
      * Anonymizer constructor.
      *
-     * @param AnnotationReader $annotationReader The annotation reader service
+     * @param AnnotationReader   $annotationReader   The annotation reader that should be used
+     * @param PropertyAnonymizer $propertyAnonymizer The property anonymizer
      */
-    public function __construct(AnnotationReader $annotationReader)
-    {
-        $this->annotationReader = $annotationReader;
+    public function __construct(
+        AnnotationReader $annotationReader,
+        PropertyAnonymizer $propertyAnonymizer
+    ) {
+        $this->annotationReader     = $annotationReader;
+        $this->propertyAnonymizer   = $propertyAnonymizer;
     }
 
     /**
-     * Anonymize the object passed
+     * Anonymizes the given object which should contain the @see Anonymize annotations.
      *
-     * @param object $object The object that is being anonymized
+     * @param object $object The object to anonymize.
      *
-     * @todo implement function
+     * @throws InvalidArgumentException
+     * @throws ReflectionException
      *
      * @return void
      */
-    public function anonymize($object)
+    public function anonymize(/*object */$object)
```

Comment:
What's with the commented out `object`?

Reply:
See @Garrcomm comment on the same question from me